CDC-Data-Reconciliation-Backend/server.py

```diff
@@ -216,6 +307,17 @@ def fetch_reports_from_db(report_id: int):
         print(f"Database error: {e}")
         return None
 
+def insert_report(createdDate, creationTime, noOfDiscrepancies):
+    cur.execute("INSERT INTO Reports (CreatedAtDate, TimeOfCreation, NumberOfDiscrepancies) VALUES (?, ?, ?)", (createdDate, creationTime, noOfDiscrepancies))
```

Comment:
You should use sql statements such as TIME('now') and DATE('now') instead of creating them in python.